### ISSUE TITLE
Leaflet Version 1.1: Map tiles not rendering in browser and rendering in RStudio viewer

### ISSUE THREAD

**Issue by granthanham** [rstudio/leaflet#386]
Map tiles are visible in the viewer but when exported (Export -> Save as Web Page) the resulting html will not render tiles when opened in I.E., Firefox, Chrome. Will do so with version 1.01. 

Minimum example (then export and open in windows based browsers):
library(leaflet)
m = leaflet() %>% addTiles()
m

FYI: This library is fantastic and the new features are very much appreciated.

RStudio Version 1.0.136

sessionInfo()
R version 3.3.2 (2016-10-31)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

locale:
[1] LC_COLLATE=English_New Zealand.1252  LC_CTYPE=English_New Zealand.1252    LC_MONETARY=English_New Zealand.1252 LC_NUMERIC=C                        
[5] LC_TIME=English_New Zealand.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] leaflet_1.1.0

**Comment by bhaskarvk** [comment]
@jcheng5 This is due to https://github.com/rstudio/leaflet/pull/343. When rendering locally the URL now becomes `file:///`.

@granthanham Try using one of the 3 party provider tiles for now.
https://rstudio.github.io/leaflet/basemaps.html

**Comment by StewartRoss** [comment]
When using the saveWidget function from htmlwidgets (latest version, 0.8) to save leaflet output to HTML opening  the resultant file now causes IE to hang (in the cases I tried). These were outputs which had worked perfectly prior to the install of leaflet 1.1. At work I am limited to IE, but at home I found that Edge also crashed, whereas in Chrome the browser did not hang but the map tiles were not rendered.

In the code generating the map I removed everything except the addtiles part of the map. Same result with IE 11 - the file generated by saveWidget hung IE. If other tabs were open these were still functioning, but the browser could not be closed except by force via task manager.

simplified map code:
     m = leaflet(PlotData_SP_LL) %>% 
          addTiles(attribution = attribution, group = "Open StreetMap")

Example of file saved by saveWidget which results in crash of IE and Edge:
[crashtest.zip](https://github.com/rstudio/leaflet/files/801018/crashtest.zip)
